app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Invalid format")
+    md, source_path, destination_path = command.split()
```

Comment:
u call command.split() twice
don't do it
create separated variable for command.split() and use it when u need it